Add Goosepaper Home Assistant Add-on (#271)
reHackable/awesome-reMarkable: README.md

- [ePUB to reMarkable PDF](https://github.com/suntorytimed/epub_to_remarkable) - Self hostable web service for turning an EPUB into a reMarkable optimised PDF.
- [Funcky reMarkable Exporter](https://github.com/simonbaudart/Funcky.Remarkable.Exporter) - Export notes from a reMarkable Tablet to File System and External Services.
- [Goosepaper](https://github.com/j6k4m8/goosepaper): Deliver prettily-formatted RSS feeds, news articles, Wikipedia articles-of-the-day, and more to your reMarkable tablet.

> - [Goosepaper Home Assistant Add-on](https://github.com/Smengerl/goosepaper-addon) - Home Assistant add-on wrapping Goosepaper: scheduled, personalized newspaper PDFs (RSS feeds, Wikipedia, weather, logic puzzles, daily comics) delivered to your reMarkable, configured and run entirely inside Home Assistant.

Reviewer: ```suggestion
- [Goosepaper Home Assistant Add-on](https://github.com/Smengerl/goosepaper-addon) - Goosepaper add-on for Home Assistant.
```
